geojson_modelica_translator/system_parameters/system_parameters.py

```diff
@@ -71,13 +70,13 @@ def __init__(self, filename=None):
         :param filename: string, (optional) path to file to load
         """
         # load the schema for validation
-        with open(os.path.join(os.path.dirname(__file__), "schema.json"), "r") as f:
+        with open((Path(__file__).parent / "schema.json"), "r") as f:
             self.schema = json.load(f)
```

Comment:
You can use Path `read_text` method to read the file. Both code will work but this is technically using the OOP goodies of Path.

```suggestion
        schema = Path(__file__).parent / "schema.json"
        self.schema = json.loads(schema.read_text())
```